style.css

```diff
@@ -0,0 +1,511 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@500;700&display=swap');
+
+body {
```

Comment:
now you have your scss setted up , transfer all the color codes to seperate variables
transfer flex properties to mixins
for eg: 
@mixin flex-col {
 display: flex;
 flex-direction: column;
} 

now everywhere this mixin will replace your two lines of code like this
.any-class {
@include flex-col;
}